js/main.js

```diff
@@ -1,141 +0,0 @@
-// API key set globally so that it is easily edited if needed
-const apiKey = '7e8e7a9139e4f09fb253db0d8ddce620';
-
-// Zip value set globally to be edited later by user input
-let zip = ''
-
-
-// Values set globally to be edited later with stateValue()
-let weatherConditions = [
-    {
-        city: '',
-        kTemp: '',
-        fahrenheitTemp: '',
-        celsiusTemp: '',
-        condition: '',
-        icon: '',
-
-    }
-]
-
-// Gets values from API
-// If zip causes API to return an error
-// catch - alert message will run
-async function getWeather() {
-    try {
-        const response = await axios.get(`https://api.openweathermap.org/data/2.5/weather?zip=${zip},us&appid=${apiKey}`)
-
-        stateValues(response.data)
-
-        initializeContent()
-    } catch {
-        alert('Enter a valid zip.\nOr else...')
-
-    // Reloads page if zip invalid
-        location.reload()
-    }
-
-};
-
-
-function createLayout(parentEl, tag, text, className, idName,) {
```

Comment:
Might be better to set the params to `{ parentEl, tag, text, className, idName }` to kill two birds with one stone. You will not care about ordering when things are passed in in the other calls and you also get to omit them whenever you want. We can talk about it later on but just a heads up.

Reply:
Good function though.